docs/process.md

```diff
@@ -0,0 +1,100 @@
+## Process
+
+Process models a set of manufacturing operations to be executed
+(i.e. [Tasks](task.md)).
+Process references Process Definition that specifies how to perform the work in detail.
+
+Process is not executed directly,
+but its constituent Tasks are executed by concrete agents.
+
+### Example #1 - Creating Process   
+
+A party requesting unloading supplies creates Process
+that specifies an existing Process Definition and some work parameters:
+
+```json
+{
+    "id": "urn:ngsi-ld:Process:company-xyz:nc847rc8347nc3",
+    "type": "Process",
+    "isDefinedBy": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:ProcessDefinition:company-xyz:unloading-supplies"
+    },
+    "workParameters": {
+        "type": "Property",
+        "value": {
+            "truckId": 838
+        }
+    },
```

Comment:
Another addition from the SAG use case. We would like to integrate the `"riskReport"` property to specify a link to a report of the risk analysis. This is necessary to guarantee traceability of the process. One example can be as follows:

```
"riskreport": {
        "type": "Property",
        "value": "link",
        "observedAt": "2020-12-01T11:23:19Z"
    },
```

Reply:
We agreed to add `outputParameters`, like in Task, to address such needs.